tests/test_wave_01.py

```diff
@@ -143,12 +144,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][-1]["title"] == movie_to_watch["title"]
```

Comment:
Great assert! However, we don't know for sure how `add_to_watched` is going to add a movie to the watched list, it's not guaranteed that it'll be at the end. Instead, a better assert would be `assert movie_to_watch in updated_data["watched"]`